Formula/ttyd.rb

```diff
@@ -0,0 +1,22 @@
+class Ttyd < Formula
+  desc "Command-line tool for sharing terminal over the web"
+  homepage "https://github.com/tsl0922/ttyd"
+  url "https://github.com/tsl0922/ttyd/archive/1.0.0.tar.gz"
+  sha256 "6fae4e932280f0ca430ea834f88282673351a321535acbe3725e995e83857138"
+  head "https://github.com/tsl0922/ttyd.git"
+
+  depends_on "cmake" => :build
```

Comment:
According to [CI failure](https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/8400/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install_ttyd/) we need to

```ruby
depends_on "pkg-config" => :build
```

I didn't catch that locally because I have it installed.